src/unistd.rs

```diff
@@ -2629,7 +2629,7 @@ pub struct User {
     /// Group ID
     pub gid: Gid,
     /// User information
-    #[cfg(not(target_os = "android"))]
+    #[cfg(not(all(target_os = "android", target_pointer_width = "32")))]
```

Comment:
This looks like an extra change that was not mentioned in the commit message.  Could you please mention it?